mu4/palette/qml/MuseScore/Palette/PalettesWidget.qml

```diff
@@ -67,37 +69,45 @@ Item {
 
         anchors {
             top: parent.top
-            topMargin: 12
             left: parent.left
             leftMargin: 12
             right: parent.right
             rightMargin: 12
         }
 
-        onAddCustomPaletteRequested: paletteTree.insertCustomPalette(0);
+        onAddCustomPaletteRequested: paletteTree.insertCustomPalette(0, paletteName);
     }
 
-    ToolSeparator {
-        id: separator
-        orientation: Qt.Horizontal
+    StyledTextLabel {
+        id: searchHint
+
         anchors.top: palettesWidgetHeader.bottom
-        width: parent.width
+        anchors.topMargin: 26
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        text: qsTr("Start typing to search all palettes")
```

Comment:
Please use our translator with the module here and elsewhere. qsTrc

Reply:
When using `qsTr`, when `.ts` files are generated (`lupdate` runs), the file name is written as the context.
This is inconvenient, the translated lines are grouped by files names, if you rename the file, the translation will be lost, if you move the line to another file, the translation will be lost, etc.

And in Qml and cpp, we explicitly set the translation context. Usually, the name of the module is used as the context. (now an explicit modular system, so this is easy to do).
In Qml, you can use `qsTranslate` for this
`qsTrc` is just a short alias (not standard, but accepted for us)

Reply:
fixed in all places